### ISSUE TITLE
New Bug Report: G2 HDI reading error

### ISSUE THREAD

**Issue by jiaguozhang** [slsdetectorgroup/slsDetectorPackage#537]
##### *Describe the bug

When starting G2 server (developer), the server could not start as the HDI ID detected are not compatible. As shown below. It dose not occur every time, but it has been seen for quite a few time. The accuracy range of the detected ID should be enlarged to avoid this error?

INFO: Checking module type
ERROR: Wrong Module attached! Expected 536, 683, 704, 723 or 747 for Gotthard2, got 530

##### Distribution: 

Any

##### Detector type or Receiver: 

G2

##### Software Package Version: 

developer, to be released version

##### Priority:

high

##### Expected behavior

no error

##### To Reproduce

just start the server from a module...

**Comment by thattil** [comment]
Yes, the tolerance is currently at 5. I will increase it for G2 to 10.
